.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+### What was wrong?
+
+### How was it fixed?
+
+### To-Do
```

Comment:
The intention looks like a checklist but TODO sounds like there is leftover task after this PR.

Reply:
This is a good place for someone to say if there is something that is going to be done in follow up PR's.

Reply:
Makes sense. But then it's not the right place for conventional commits. It should have already done, not to be done in subsequent PRs.

Reply:
Having `conventional commits` makes it look like a checklist.

Reply:
It is like a to-do list, where you have things to check off

Reply:
> Having `conventional commits` makes it look like a checklist.

![image](https://github.com/user-attachments/assets/e869bcf7-805a-4328-8708-47a7df7bf5b3)

Reply:
I don't think a `done list` is a thing

Reply:
I think this section should cover things that needs to be completed before merging the PR.

For future work in future PRs, the author should create new issues instead. The todo list in the PR will fade away over time.

Reply:
> I don't think a `done list` is a thing

True that. I don't think it's a done list is required to be in the template.

I still stand that todo should become issues though.

Reply:
Missing the point. The conventional commit is not a good example for to-do list as it should be already done in each PR. We won't have a following PR to make a convention commit that current RP is missing.

Reply:
> Missing the point. The conventional commit is not a good example for to-do list as it should be already done in each PR. We won't have a following PR to make a convention commit that current RP is missing.

the `to-do` section doesn't only have to be used for followup PR's, it is a check list of things to do.

Reply:
> This is a good place for someone to say if there is something that is going to be done in follow up PR's.

So it's not for future PRs only. Then, I'd add an item to read contributing.md.

Reply:
Sure